Source/Core/Core/GeckoCodeConfig.cpp

```diff
@@ -139,6 +139,9 @@ std::vector<GeckoCode> LoadCodes(const IniFile& globalIni, const IniFile& localI
     {
       std::istringstream ss(line);
 
+      //Force ISO-8859-1 encoding, UTF-8 breaks formating
+      ss.imbue(std::locale("C"));
```

Comment:
You should probably use this instead:
```suggestion
      ss.imbue(std::locale::classic());
```

Reply:
Out of curiosity, what does it change ?

Reply:
It's a specialized class rather than doing a lookup on the "C" locale (which internally probably does just the same in the end), so it _should_ not make a difference; but thats more in line with what we have everywhere else.